packages/runtime/container-runtime/src/summary/summarizer.ts

```diff
@@ -124,12 +125,20 @@ export class Summarizer extends TypedEventEmitter<ISummarizerEvents> implements
 		};
 
 		const resolvedContainer = await loader.resolve(request);
-		const fluidObject: FluidObject<ISummarizer> | undefined = resolvedContainer.getEntryPoint
-			? await resolvedContainer.getEntryPoint?.()
-			: // eslint-disable-next-line import/no-deprecated
-			  await requestFluidObject<FluidObject<ISummarizer>>(resolvedContainer, {
-					url: "_summarizer",
-			  });
+		let fluidObject: FluidObject<ISummarizer> | undefined;
+
+		// Older containers may not have the "getEntryPoint" API
+		// ! This check will need to stay until LTS of loader moves past 2.0.0-internal.7.0.0
+		if (resolvedContainer.getEntryPoint !== undefined) {
```

Comment:
Side note - I wonder if there'd be a reasonable way to properly model the support matrix here - like have `resolvedContainer` be a union of `IContainer` from every major version back to LTS or something.  Not for this PR, but since you're dealing with this I'm mentioning it.